rsocket-transport-netty/src/main/java/io/rsocket/transport/netty/client/TcpClientTransport.java

```diff
@@ -104,13 +105,15 @@ public static TcpClientTransport create(TcpClient client) {
                 c -> {
                   if (mtu > 0) {
                     return new FragmentationDuplexConnection(
-                        new TcpDuplexConnection(c, false),
+                        new ReassemblyDuplexConnection(
+                            new TcpDuplexConnection(c, false), ByteBufAllocator.DEFAULT, true),
                         ByteBufAllocator.DEFAULT,
                         mtu,
                         true,
                         "client");
```

Comment:
This would become a little nicer if `FragmentationDuplexConnection(...)` was a sub-class of `ReassemblyDuplexConnection`

Reply:
fixed